From ec874b9ce89014ca2c80595a3653e2fe4cc7094b Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Thu, 5 Feb 2026 08:54:59 +0100
Subject: [PATCH 106/N] makesetup: make sure to link the built libpython

dependency libs might add system search paths first, so
make sure we link to libpython before that.
Otherwise we link to the system libpython if it already exists.
---
 Modules/makesetup | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Modules/makesetup b/Modules/makesetup
index 2ffc3dd..99fb22d 100755
--- a/Modules/makesetup
+++ b/Modules/makesetup
@@ -276,7 +276,7 @@ sed -e 's/[ 	]*#.*//' -e '/^[ 	]*$/d' |
 				;;
 			esac
 			rule="$file: $objs \$(MODULE_${mods_upper}_LDEPS)"
-			rule="$rule; $ccld $objs $libs \$(BLDLIBRARY) -o $file"
+			rule="$rule; $ccld $objs \$(BLDLIBRARY) $libs -o $file"
 			echo "$rule" >>$rulesf
 		done
 	done
